src/prefect/deployments/deployments.py

```diff
@@ -63,14 +63,22 @@ async def run_deployment(
     tags: Optional[Iterable[str]] = None,
     idempotency_key: Optional[str] = None,
     work_queue_name: Optional[str] = None,
-):
+    as_subflow: Optional[bool] = True,
+) -> FlowRun:
     """
-    Create a flow run for a deployment and return it after completion or a timeout.
+    Create a flow run for a deployment and return its metadata.
```

Comment:
is there a reason to remove "after completion or a timeout" here? I feel like that has been useful for those who didn't realize it waits/blocks.

Reply:
The wording of this docstring felt a little awkward, but also, I'm optimizing for the fact that so few people seem to know this function can be called _without_ blocking. So, maybe I went too far!

Reply:
@serinamarie The wording of this docstring felt a little awkward, but also, I'm optimizing for the fact that so few people seem to know this function can be called _without_ blocking. So, maybe I went too far! I'll roll that back.